provider/aws/aws.go

```diff
@@ -35,7 +37,8 @@ import (
 )
 
 const (
-	recordTTL = 300
+	DefaultAWSProfile = "default"
```

Comment:
As far as I see we don't need to expose this const. `DefaultAWSProfile` -> `defaultAWSProfile`